Commands/Fields/AddField.cs

```diff
@@ -20,37 +20,45 @@ public class AddField : SPOWebCmdlet, IDynamicParameters
     {
         [Parameter(Mandatory = false, ValueFromPipeline = true, ParameterSetName = "ListPara")]
         [Parameter(Mandatory = true, ValueFromPipeline = true, ParameterSetName = "FieldRef")]
+        [Parameter(HelpMessage = "The list object or name where this field needs to be added")]
         public ListPipeBind List;
 
-        [Parameter(Mandatory = true, ParameterSetName = "FieldRef")]
+        [Parameter(Mandatory = true, ParameterSetName = "FieldRef", HelpMessage = "Field Object that needs to be added")]
```

Comment:
ID, Field name or actual fieldobject are acceptable for FieldPipeBind parameters.